src/main/java/me/oondanomala/fpkmod/labels/input/LabelLastSidestep.java

```diff
@@ -0,0 +1,16 @@
+package me.oondanomala.fpkmod.labels.input;
+
+import me.oondanomala.fpkmod.labels.TextLabel;
+import me.oondanomala.fpkmod.movement.ParkourHandler;
+
+public class LabelLastSidestep extends TextLabel {
+    public LabelLastSidestep() {
+        super("Last Sidestep");
+    }
+
+    @Override
+    protected String getLabelText() {
+        int lastSidestep = ParkourHandler.sidestep;
+        return lastSidestep == 0 ? "WAWD" : "WAD " + Math.abs(lastSidestep) + "t";
```

Comment:
```suggestion
        return lastSidestep == 0 ? "WDWA" : "WAD " + Math.abs(lastSidestep) + "t";
```
I prefer `WDWA` over `WAWD`
Also needs to be changed in the `sidestep` javadoc